### PR TITLE
Merge pull request #1364 from dmitriysmirnovjb/master

### PR DESCRIPTION
**Pull request by dmitriysmirnovjb**
Hello,
Judging by the recent SO surveys ([2019](https://insights.stackoverflow.com/survey/2019/#development-environments-and-tools) and [2018](https://insights.stackoverflow.com/survey/2018/#development-environments-and-tools)), IntelliJ and other IDEs by JetBrains are pretty popular. So it seems adding basic information about Git Integration in these IDEs will be helpful for the community.

I tried to make it as brief as possible to make sure it does not become outdated soon.

### CODE REVIEW

File: book/A-git-in-other-environments/sections/jetbrainsides.asc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+=== Git in IntelliJ / PyCharm / WebStorm / PhpStorm / RubyMine
+
+JetBrains IDEs - IntelliJ IDEA / PyCharm / WebStorm / PhpStorm / RubyMine / etc - ship with the Git Integration plugin.
```
Comment (ben) [inline book/A-git-in-other-environments/sections/jetbrainsides.asc]:
```suggestion
JetBrains IDEs (such as IntelliJ IDEA, PyCharm, WebStorm, PhpStorm, RubyMine, and others) ship with a Git Integration plugin.
```

File: book/A-git-in-other-environments/sections/jetbrainsides.asc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+It provides the dedicated view in the IDE to work with Git and GitHub Pull Requests.
```
Comment (ben) [inline book/A-git-in-other-environments/sections/jetbrainsides.asc]:
```suggestion
It provides a dedicated view in the IDE to work with Git and GitHub Pull Requests.
```

File: book/A-git-in-other-environments/sections/jetbrainsides.asc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+.Version Control ToolWindow in JetBrains IDEs.
+image::images/jb.png[Version Control ToolWindow in JetBrains IDEs.]
+
+The integration relies on the command-line git client and requires one to be installed.
```
Comment (ben) [inline book/A-git-in-other-environments/sections/jetbrainsides.asc]:
```suggestion
The integration relies on the command-line git client, and requires one to be installed.
```

File: book/A-git-in-other-environments/sections/jetbrainsides.asc [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (ben) [inline book/A-git-in-other-environments/sections/jetbrainsides.asc]: Let's go ahead and remove this line.

**Comment by dmitriysmirnovjb** [comment]
I've just pushed the requested changes

File: book/A-git-in-other-environments/sections/jetbrainsides.asc [hunk 5]
```diff
@@ -0,0 +1,10 @@
+=== Git in IntelliJ / PyCharm / WebStorm / PhpStorm / RubyMine
+
+JetBrains IDEs (such as IntelliJ IDEA, PyCharm, WebStorm, PhpStorm, RubyMine, and others) ship with a Git Integration plugin.
+It provides a dedicated view in the IDE to work with Git and GitHub Pull Requests.
+
+.Version Control ToolWindow in JetBrains IDEs.
+image::images/jb.png[Version Control ToolWindow in JetBrains IDEs.]
+
+The integration relies on the command-line git client, and requires one to be installed.
+The official documentation is available at https://www.jetbrains.com/help/idea/using-git-integration.html
```
Comment (ben) [inline book/A-git-in-other-environments/sections/jetbrainsides.asc]: One final change, just for completeness:

```suggestion
The official documentation is available at https://www.jetbrains.com/help/idea/using-git-integration.html[].
```

**Comment by ben** [comment]
✨ Thanks!

**Comment by dmitriysmirnovjb** [comment]
Not sure if something is wrong with my changes, but for some reasons, the image on https://git-scm.com/book/en/v2/Appendix-A%3A-Git-in-Other-Environments-Git-in-IntelliJ-%2F-PyCharm-%2F-WebStorm-%2F-PhpStorm-%2F-RubyMine is not disaplayed

**Comment by ben** [comment]
That's an issue with that website, whose source code is at https://github.com/git/git-scm.com. I'd recommend you file an issue there, I've got no control over their publishing process.

**Pull request merged**